### DISCUSSION TITLE
Designs

### DISCUSSION THREAD

**Discussion by AishaKeyf** [dugsiiyeinc/dhaqsoPay#9] [Ideas]
Dhallinyaroooy!!
 wxaa yaalla 3 design oo cusub.
midba kan kale waa ka daran yhy, bal indhihiinaa idiin macalina...
kan ugu danbeeya baa ani ila shidan.

1=  https://preview.themeforest.net/item/raveli-online-payment-app-elementor-template-kit/full_screen_preview/43419505

2=  https://preview.themeforest.net/item/yakopay-online-payment-app-elementor-template-kit/full_screen_preview/34138538

3=  https://preview.themeforest.net/item/switch-payment-app-elementor-template-kit/full_screen_preview/29931491

**Comment by abdinasir-Tman** [comment]
@dugsiiyeinc/dhaqsopay-contributors

**Comment by sharafdin** [comment]
> Dhallinyaroooy!! wxaa yaalla 3 design oo cusub. midba kan kale waa ka daran yhy, bal indhihiinaa idiin macalina... kan ugu danbeeya baa ani ila shidan.
> 
> 1= https://preview.themeforest.net/item/raveli-online-payment-app-elementor-template-kit/full_screen_preview/43419505
> 
> 2= https://preview.themeforest.net/item/yakopay-online-payment-app-elementor-template-kit/full_screen_preview/34138538
> 
> 3= https://preview.themeforest.net/item/switch-payment-app-elementor-template-kit/full_screen_preview/29931491

cool waye runtii. aniga kan ugu horeeya aad ii qanciyay waa simple waliba modern ah.

**Reply by shiinedev** [reply]
saddexda design ba waa miisanka culus aniga kan labad ayan wata oo design ahan aad ii qanciye

**Comment by Nasrah-muse** [comment]
> Dhallinyaroooy!! wxaa yaalla 3 design oo cusub. midba kan kale waa ka daran yhy, bal indhihiinaa idiin macalina... kan ugu danbeeya baa ani ila shidan.
> 
> 1= https://preview.themeforest.net/item/raveli-online-payment-app-elementor-template-kit/full_screen_preview/43419505
> 
> 2= https://preview.themeforest.net/item/yakopay-online-payment-app-elementor-template-kit/full_screen_preview/34138538
> 
> 3= https://preview.themeforest.net/item/switch-payment-app-elementor-template-kit/full_screen_preview/29931491

cool weye eng dhamaantood  kan ugu kowaad baa ila haboon kuwa kale waa design-nadii hore 👍

kan kawarama sidoo kale https://html.designingmedia.com/repay/?storefront=envato-elements

**Reply by sharafdin** [reply]
aniga ahaan waan fasaxay design kaan waa top.

labadaan kala codeeya kadib wana ku soconeenaa poll ayaan ka sameynaa Insha'Allah!
